src/compiler/emitter.ts

```diff
@@ -7885,5 +7883,22 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                 emitSkipped = writeDeclarationFile(declarationFilePath, sourceFiles, isBundledEmit, host, resolver, emitterDiagnostics) || emitSkipped;
             }
         }
+
+        /**
+         * Write the file names for the emitted files to the console, don't actually compile and emit.
+         * This helps build tools to determine if they should invoke a full compilation.
+         */
+        function emitNullFile({ jsFilePath, sourceMapFilePath, declarationFilePath}: { jsFilePath: string, sourceMapFilePath: string, declarationFilePath: string },

```

Comment:
I would use type name `EmitFileNames` for the first parameter here. (and for `emitFile` too)